authors/apps/authentication/urls.py

```diff
@@ -12,4 +13,5 @@
     path('users/login/twitter/', TwitterAuthAPIView.as_view(), name='twitter-auth'),
     path('users/login/google/', GoogleAuthAPIView.as_view(), name='google-auth'),
     path('users/login/facebook/', FacebookAuthAPIView.as_view(), name='facebook-auth'),
+    path('users/<token>/', AccountActivateAPIView.as_view(), name='activate'),
```

Comment:
@codjoero I feel that writing the namespace here as activate-account would be more descriptive than just activate. What do you think?

Reply:
@missvicki , great suggestion, adding...